### DISCUSSION TITLE
Cursor stuck MacBook osu lazer with HUION H430P

### DISCUSSION THREAD

**Discussion by dsdt1823** [ppy/osu#23548] [Q&A]
So I recently bought this HUION H430P, and it works perfectly fine at running osu for a few days. But after I installed a HUION Mac app stuff and played around with it, my osu suddenly stop working fine. The cursor is stuck at the top left while with my pen. I've tried multiple ways such as following this forum [community/forums/topics/590367](https://osu.ppy.sh/community/forums/topics/590367) but I don't know where the enable window ink in mac is. i found out that turning off high precision mouse or confined mouse to window kind of helped the pen but it ruined the sensitivity. I've also reinstalled both the HUION software and osu but it still did not work. can someone help me?

**Comment by MishterKirby** [comment]
Try disabling "High-precision mouse" in osu!'s settings, a lot of tablets seem to have that issue with their default drivers

**Reply by dsdt1823** [reply]
but it will disable the sensitivity and its pretty annoying. is there like another way to do it?

**Reply by MishterKirby** [reply]
What options do the macOS drivers have? I'm only familiar with Huion's Windows drivers
